### PR TITLE
Add Pokemon to list of uncountables

### PR DESCRIPTION
**Pull request by hugomd**
Hey! 👋 Thanks for this awesome module 🙏

The plural of Pokémon is [not Pokémons](https://en.wikipedia.org/wiki/Portal:Pok%C3%A9mon), so I've added it to the list of uncountables. 

✌️

### CODE REVIEW

**Comment by blakeembrey** [comment]
Happy to merge this, but should we also handle it with the "é"?

**Comment by hugomd** [comment]
@blakeembrey I've added an extra case for "é", and some tests, although I'm not sure if the test cases are in the right place.. If they're not, happy to fix, other than that should be good to merge 👍 

Thanks for making this project 🙏

File: pluralize.js [hunk 1]
```diff
@@ -476,6 +476,7 @@
     'wildebeest',
     'wildlife',
     'you',
+    '/pok(e|é)mon/',
```
Comment (blakeembrey) [inline pluralize.js]: This shouldn't be working - it's a string...

Comment (blakeembrey) [reply pluralize.js]: Yeah, this definitely isn't working - the tests are in the wrong place. `SINGULAR_TESTS` are for "odd plurals" to singular only (doesn't test singular to plural).

Comment (hugomd) [reply pluralize.js]: Very good point, sorry about that!

**Pull request merged**